sc/src/main/webapp/WEB-INF/jsp/tiles/views/experiment-page.jsp

```diff
@@ -1,11 +1,17 @@
 <%@ page contentType="text/html;charset=UTF-8" %>
 <%@ include file="experiment-page/experiment-description.jsp" %>
 
+<link rel="stylesheet" href="http://localhost:8080/gxa/sc/resources/css/popup-feedback.css" type="text/css" media="all">
+<link rel="stylesheet" href="http://ebi.emblstatic.net/web_guidelines/EBI-Icon-fonts/v1.3/fonts.css" type="text/css" media="all">
```

Comment:
Same comment as before, you must add CSS resources in the `base.jsp`, otherwise we will lose track of which CSS we use there. Saying that, we have already added the EBI icons v1.2. You cannot add v1.3 as well as they do not play nicely with each other. So try to pick an icon that is available in 1.2.

We do have another story to migrate to 1.3 (https://www.pivotaltracker.com/story/show/163226652), but it's not as straighforward as it seems, as 1.3 has a lot of breaking changes. Therefore, a migration should be done as part of a separate story.